modules/setting-up-bastion-for-sno.adoc

```diff
@@ -0,0 +1,172 @@
+// This module is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+:_content-type: PROCEDURE
+[id="setting-up-bastion-for-sno_{context}"]
+= Setting up basion for {sno} with {ibmpowerProductName}
+
+Prior to installing {sno} on {ibmpowerProductName}, you must set up bastion. Setting up a bastion server for {sno} on {ibmpowerProductName} requires the configuration of the following services: 
+
+* PXE is used for the {sno} cluster installation. It requires the following services to be configured and run:
+** DNS to define api, api-int and *.apps
+** DHCP service to enable PXE and assign IP to {sno} node
+** HTTP to provide ignition and RHCOS rootfs image
+** TFTP to enable PXE
+* You must install `dnsmasq` to support DNS, DHCP and PXE, httpd for HTTP.
+Use the following procedure to configure a bastion server that meets these requirements.
```

Comment:
Need a space here between lines 16 and 17. Content is on the same line as the previous bullet point. See rendering for what I'm talking about if you need to.

Reply:
Ack. Updated the PR. Thanks.